src/linear_algebra/alternating.lean

```diff
@@ -37,6 +43,57 @@ using `map_swap` as a definition, and does not require `has_neg N`.
 * `alternating_map.coe_smul`
 -/
 
+-- TODO: move
+section to_move
+
+namespace tactic
+namespace interactive
+
+open lean.parser
+open interactive
+
+/-- Focus on the first `n` goals. -/
+meta def first_n_goals : parse small_nat → itactic → tactic unit
```

Comment:
do you think `work_on_goal` could be generalised to include this use case?

Reply:
Not really, because `work_on_goal n` has semantics "work on the goal, and them emit `old_goals[:n] ++ new_goals ++ old_goals[n + 1:]`. If you're working on multiple goals, there's no sensible choice for where to insert the new goals.